qkeras/qconvolutional.py

```diff
@@ -155,8 +156,11 @@ def get_config(self):
   def get_quantizers(self):
     return self.quantizers
 
+  def get_prunable_weights(self):
+    return [self.kernel]
 
-class QConv2D(Conv2D):
+
+class QConv2D(Conv2D, PrunableLayer):
```

Comment:
Have you considered ClusterableLayer as well?

Reply:
I have :-) But I thought that you would prefer that as a separate PR. I can add it to this one if you want?